<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qnx::Internal::BlackBerryCreatePackageStepConfigWidget</class>
 <widget class="QWidget" name="Qnx::Internal::BlackBerryCreatePackageStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="signPackages">
     <property name="text">
      <string>Sign packages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="signPackagesWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>CSK password:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="cskPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Keystore password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="keystorePassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QCheckBox" name="savePasswords">
          <property name="toolTip">
           <string>Note: This will store the passwords in a world-readable file.</string>
          </property>
          <property name="text">
           <string>Save passwords</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showPasswords">
        <property name="text">
         <string>Show passwords</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="developmentMode">
     <property name="text">
      <string>Package in development mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="developmentModeWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="qtLibrary"/>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="qtLibraryLabel">
          <property name="text">
           <string>/accounts/devuser/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="qtLibraryPath"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="deployNowButton">
        <property name="text">
         <string>Deploy Now</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="deployErrorPixmap">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../projectexplorer/projectexplorer.qrc">:/projectexplorer/images/compile_warning.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="deployErrorLabel">
        <property name="text">
         <string>No device configured, cannot deploy.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLabel" name="qtLibraryExplanationLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../projectexplorer/projectexplorer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>signPackages</sender>
   <signal>toggled(bool)</signal>
   <receiver>signPackagesWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>developmentMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>developmentModeWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>signPackages</sender>
   <signal>toggled(bool)</signal>
   <receiver>developmentModeWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>developmentMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>signPackagesWidget</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
